recipes/pywin32/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "pywin32" %}
+{% set version = "220" %}
+{% set hash_type = "sha1" %}
+{% set hash = "88ba6a31c077a9e1b49b59dc215ddadc6aa5e3d6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://downloads.sourceforge.net/project/{{ name }}/{{ name }}/Build {{ version }}/{{ name }}-{{ version }}.zip
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup{{ '3' if py3k else '' }}.py install --record=record.txt
+  skip: True  # [not win]
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
+
+requirements:
+  build:
+    - python  # [win]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py>=35]
+  run:
+    - python
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py>=35]  
+
+test:
+  imports:
+    # list obtained from: http://timgolden.me.uk/pywin32-docs/win32_modules.html
+    # skipped:
+    #   - _winxptheme  # private module
+    #   - wincerapi  # interface to the win32 CE Remote API
+    - mmapfile
+    - odbc
+    - perfmon
+    - pywintypes
+    - servicemanager
+    - timer
+    - win2kras
+    - win32api
+    - win32clipboard
+    - win32console
+    - win32cred
+    - win32crypt
+    - win32event
+    - win32evtlog
+    - win32file
+    - win32gui
+    - win32help
+    - win32inet
+    - win32job
+    - win32lz
+    - win32net
+    - win32pdh
+    - win32pipe
+    - win32print
+    - win32process
+    - win32profile
+    - win32ras
+    - win32security
+    - win32service
+    - win32transaction
+    - win32ts
+    - win32wnet
+
+about:
+  home: https://sourceforge.net/projects/pywin32
+  license: PSF
```

Comment:
I guess they are not clear about this either. 😕

Also seems to include other libraries that have various licenses. Mainly BSD and LGPL. Also includes SWIG, which is GPL, but I don't think that is a concern as one doesn't link to SWIG. Plus SWIG has an exception for this case.